packages/expo-video/src/VideoView.tsx

```diff
@@ -84,3 +90,43 @@ function getPlayerId(player: number | VideoPlayer): number | null {
   }
   return null;
 }
+
+/**
+ * Returns a shared object, which is automatically cleaned up when the component is unmounted.
+ */
+// TODO: when SharedObject type is added make T extend it
```

Comment:
Add it inside the `/** */` comment, otherwise I think it wouldn't be included in the generated docs